### DISCUSSION TITLE
Is there a way to add button actions?

### DISCUSSION THREAD

**Discussion by joshbedo** [Comcast/react-data-grid#2576] [Q&A]
I want to add edit/delete buttons to the end of each row and I was wondering if there is any way to make a specific column with the buttons frozen and not highlightable?

<img width="1203" alt="Screen Shot 2020-04-21 at 12 35 11 PM" src="https://user-images.githubusercontent.com/607239/79889863-9daab980-83cc-11ea-9c2e-bc740cbc44d9.png">

**Comment by iamravisingh** [comment]
@joshbedo you can achieve it .. while initial render you can append a row and you can use the **isLastFrozenColumn** flag to make the last column frozen
